specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/stable/2018-11-20/examples/createOrUpdateGuestConfigurationAssignment.json

```diff
@@ -55,9 +54,8 @@
               }
             ],
             "configurationSetting": {
-              "rebootIfNeeded": "False",
+              "rebootIfNeeded": false,
               "actionAfterReboot": "ContinueConfiguration",
-              "configurationModeFrequencyMins": 15,
```

Comment:
Same question as above

Reply:
Added back in. I was trying to fix the validation error.